frontend/src/types/index.ts

```diff
@@ -75,20 +75,22 @@ export type Location = {
   tag?: Tag;
 };
 
-export type Passenger = {
-  startTime: string;
-  endTime: string;
-  name: string;
-  pickupLocation: string;
-  pickupTag: string;
-  dropoffLocation: string;
-  dropoffTag: string;
-  needs: string;
-};
-
 export type TableValue = {
   data: string | null;
   tag?: string;
   buttonHandler?: () => void;
   ButtonModal?: () => JSX.Element;
 };
+
+export type RealRide = {
```

Comment:
Could you think of a better name for this than "RealRide"? Is it possible to just call this Ride, replacing the original type that is already there? Or will that cause errors?

Reply:
I didn't want to cause errors with the existing code using `Ride`, but yes, `RealRide` is not a great name. I'll message the creator of `Ride` type to see if we can get a better name for the two of them.

Reply:
From what I can tell, the `Ride` type was defined but was never used. I'm going to switch `RealRide` to `Ride` and if the original `Ride` ever needs to be used, I think it's best that the type is defined locally because adding a ride is only done in one place. If a different approach is needed, the names can always be changed later.